modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -127,7 +127,10 @@ def _str_map(func_name):
     def str_op_builder(df, *args, **kwargs):
         """Apply specified function against `str` accessor of the passed frame."""
         str_s = df.squeeze(axis=1).str
-        return getattr(pandas.Series.str, func_name)(str_s, *args, **kwargs).to_frame()
+        res = getattr(pandas.Series.str, func_name)(str_s, *args, **kwargs)
```

Comment:
This operation may return a DataFrame that does not have this method. Previously, only the Series returned.